.circleci/config.yml

```diff
@@ -2200,13 +2209,6 @@ workflows:
       - binary_linux_build:
           name: binary_linux_manywheel_3_7m_cu101_devtoolset7_nightly_build
           build_environment: "manywheel 3.7m cu101 devtoolset7"
-          filters:
```

Comment:
why are we removing these filters?

Reply:
nvm

Reply:
This is a temp change just to test that it will work for binary builds